config.json

```
@@ -852,6 +860,14 @@
         "prerequisites": [],
         "difficulty": 6
       },
+      {
+        "slug": "transpose",
+        "name": "Transpose",
+        "uuid": "59b5d30b-62c1-4465-a44d-485c45f4aac2",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
```

```suggestion
        "difficulty": 4
```
It's not a trivial exercise, let's increase the difficulty